### PR TITLE
Add Python 3 Debian package

### PR DESCRIPTION
**Pull request by samdbmg**
This adds a `python3-nodefacade` package to the Debian package scripts. Not that it's a huge amount of use, since `python3-nmoscommon` doesn't pull its dependencies in properly, and `python3-pybojour` is just broken.
Will build on Jenkins if configured on one of the `pbuilder-xenial-amd64` slaves - some of the `pbuilder-xenial` ones (notably jenkins-slave-acbuild.rd...) don't have python3-setuptools.

### CODE REVIEW

**Review by andrewbonney — COMMENTED** [review]
LGTM. Have you tried installing it at all having built a package locally?

**Comment by samdbmg** [comment]
I've installed it locally and that all seems to work; but given `python3-pybonjour` is broken and there's no `python3-avahi` it's difficult to meaningfully test. Should this one go on the back burner pending that stuff being sorted? Shall I delete the PR or leave it open for now?

**Comment by andrewbonney** [comment]
I'd leave it open. I've added a needs work label even though it's technically work in the dependency: https://github.com/bbc/nmos-common/pull/8. Lack of python3-avahi is useful to know as it adds more weight to potentially moving away from that, although it may be that the code is compatible and the packaging is just missing.

File: debian/control [hunk 1]
```diff
@@ -2,12 +2,19 @@ Source: python-nodefacade
...
 Section: python
 Priority: optional
-Build-Depends: python-all (>= 2.6.6-3), debhelper (>= 7), dh-python, python-setuptools, dh-systemd, dh-apache2
+Build-Depends: python-all (>= 2.6.6-3), python3-all, debhelper (>= 7), dh-python, python-setuptools, python3-setuptools, dh-systemd, dh-apache2
 Standards-Version: 3.9.7
 
 Package: python-nodefacade
 Architecture: all
-Depends: ${misc:Depends}, ${python:Depends}, python-systemd
-Recommends: lldpd, python-nmoscommon, ips-reverseproxy-common, python-mdnsbridge
```
Comment (simonrankine) [inline debian/control:11]: For annoying reasons the joint-ri currently needs python-nmoscommon to be in recommends not depends... I'm aware this is an issue but until I have time to sort it out it needs to stay where it is please :)

**Review by simonrankine — CHANGES_REQUESTED** [review]

**Pull request closed without merging**